mumo/test.md

```diff
@@ -0,0 +1,38 @@
+# MuMoBrain
+MuMo is a multimodal project in which MRI (resting-state, diffusion, spectroscopy), MEG, EEG, and NPA data was collected from healthy human subjects to explore the multimodal correlates of cognitive functioning through the framework of multilayer networks. 
+
+## mumo_script_clean_v1.m
+This script is used to convert pre-processed neuroimaging data from different modalities (fMRI, DWI, MEG) into connectivity matrices, and to perform monolayer network analyses on and construct supra-adjacency matrices for multilayer analyses from these matrices. It consists of four parts:
+
+1.	Calculating connectivity
+2.	Randomizing data
+3.	Supra-adjacency matrices
+4.	Network measures
+
+A more detailed breakdown of these four parts is provided below. The following functions, to be found in the /functions folder, are used in the script:
+
+* pli_matteo.m
+* fft_filt_BNA.m
+* remove_empty_regions.m
+* kruskal_algorithm.m
+
+It further uses several functions from the [Brain Connectivity Toolbox](https://sites.google.com/site/bctnet/), version 2019_03_03.
+
+
+### Part 1: Calculating connectivity
+In Part 1, functional connectivity for MEG and fMRI timeseries is determined and a DWI structural connectivity matrix is constructed. Empty regions (i.e. missing signal) are then removed from these matrices, data is normalized, and minimum spanning trees are constructed.
```

Comment:
Perhaps indicate what input is used for the construction of the structural matrix? As in: you're not doing the tractography here.

And small reformulations: 
DWI = dMRI
data is = matrices are